relay-common/src/glob3.rs

```diff
@@ -12,6 +12,43 @@ fn is_match(globs: &[Regex], message: &[u8]) -> bool {
     globs.iter().any(|regex| regex.is_match(message.as_ref()))
 }
 
+/// A singular glob-pattern.
+#[derive(Debug, Clone, PartialEq, Default)]
```

Comment:
Does default make sense here? What is a default `GlobPattern`, the empty String?

`Serialize` will fail if the inner is `[]`.

Reply:
right, clear oversight, thank you!